master/security/http-methods.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Using HTTP methods and paths in policy rules
+---
+
+### Big picture
+
+Use Calico network policy for Istio-enabled apps to restrict ingress traffic that match HTTP methods or paths. 
+
+### Value
+
+Customer-facing applications like web apps can benefit from selective restrictions on ingress traffic (for example, HTTP GET requests). 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **NetworkPolicy** 
+- **GlobalNetworkPolicy** with http match criteria to restrict traffic using:
```

Comment:
This structure makes it seem like these are only supported on GNP, when actually they are supported on both NP and GNP.

Reply:
Fixed.

Reply:
Fixed